### PR TITLE
[res] Add Net_Conv_Add_Mul_004 TFLite recipe

### PR DESCRIPTION
**Pull request by jinevening**
This adds Net_Conv_Add_Mul_004 TFLite recipe.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/8597
Draft PR: https://github.com/Samsung/ONE/pull/8602

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Net_Conv_Add_Mul_004/test.rule [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To check mixed-precision quantization.
+# Conv is int16, and others u8. Quantize Ops are inserted before/after Conv.
```
Comment (jinevening) [inline res/TensorFlowLiteRecipes/Net_Conv_Add_Mul_004/test.rule:2]: This is what this rule wants to check.

File: res/TensorFlowLiteRecipes/Net_Conv_Add_Mul_004/test.qconf.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_Conv_Add_Mul_004/test.qconf.json:1]: To express this network for quantization test, how about using name
`Quant_Conv_Add_Mul_001`?

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by llFreetimell — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
@jinevening , I've changed commit message with `Quant_Conv_Add_Mul_004`

**Pull request merged**